docs/aerodromes/classc/Brisbane.md

```diff
@@ -29,6 +29,74 @@ There are three SMC positions at Brisbane which are responsible for various part
   <figcaption>YBBN Maneuvering Area Responsibility</figcaption>
 </figure>
 
+## Helicopter Operations
+Brisbane has one helipad located within the general aviation apron, most rescue helicopters are based within the general aviation maintenance area and will depart from the **H2 taxiway** to save time when operating.
+
+#### Waypoints
+VFR and IFR helicopters are normally processed via two visual waypoints north and south of the field, when in VMC conditions. IFR helicopters in IMC conditions may conform to fixed wing ops and be processed via the BN (RADAR) SID from an appropriate runway, unless a visual departure is acceptable. These waypoints are used for **departures** and **arrivals**, depending on weather and traffic conditions.
+
+
+| Direction               | Waypoint       | Name        |
+| ------------------ | -------------- | ---------------- |
+| North      | BLHS   | Bald Hills          |
+| South      | MBHM   | Manly Boat Harbour          |
+| East      | MBHM   | Manly Boat Harbour          |
+| West      | BLHS   | Bald Hills          |
+
+#### Departures
+
+Refer to [Maneuvering Area Responsibility](#maneuvering-area-responsibility) to determine which SMC or ADC position (when non-standard positions are online) is responsible for managing helicopter arrivals and departures.
```

Comment:
We'll need to decide which ADC is responsible for each part of the aerodrome, since ADC doesn't own any airspace by default. This might need to be coordinated with the TMA or we might need to give ADC some airspace to manage it themselves. @alphadelta332  thoughts?